<launch>
  <arg name="uav" default="uav1" />
  <node name="pathfinder" pkg="pathfinder" type="pathfinder" output="screen">
    <rosparam file="$(find pathfinder)/config/pathfinder.yaml" command="load" />
    <remap from="~odom_in" to="/$(arg uav)/odometry/odom_main" />
    <!-- <remap from="~octomap_in" to="/$(arg uav)/???" /> -->
  </node>

</launch>
